src/main/java/ch/cern/eam/wshub/core/services/workorders/ChecklistService.java

```diff
@@ -22,4 +21,10 @@
     @Operation(logOperation = INFOR_OPERATION.WO_CHECKL_FOLLOWUPWO_C)
     Long createFollowUpWorkOrders(InforContext context, Activity activity) throws InforException;
 
+    //TODO: FIX @OPERATION
+    //    @Operation(logOperation = INFOR_OPERATION.WO_CHECKL_U)
+    WorkOrderActivityChecklistSignatureResponse eSignWorkOrderActivityChecklist(InforContext context, WorkOrderActivityCheckListSignature workOrderActivityCheckListSignature)
+            throws Exception;
+
+    WorkOrderActivityChecklistSignatureResult[] getSignatures(InforContext context, String workOrderCode, String activityCode) throws Exception;
```

Comment:
1. Missing Operation annotation;
2. These should throw `InforException`s.

Reply:
For 2. - both methods call the marshal method in DateAdapter which throws Exception. Should I change it to 
`WorkOrderActivityChecklistSignatureResponse eSignWorkOrderActivityChecklist(InforContext context, WorkOrderActivityCheckListSignature workOrderActivityCheckListSignature)
            throws InforException, Exception;

    WorkOrderActivityChecklistSignatureResult[] getSignatures(InforContext context, String workOrderCode, String activityCode) throws InforException, Exception;`